livekit-ffi/protocol/ffi.proto

```diff
@@ -67,17 +67,18 @@ message FfiRequest {
     UpdateLocalNameRequest update_local_name = 10;
     GetSessionStatsRequest get_session_stats = 11;
     PublishTranscriptionRequest publish_transcription = 12;
+    GetRoomSidRequest get_room_sid = 13;
```

Comment:
Let's use this [way](https://github.com/livekit/rust-sdks/pull/342#discussion_r1619337144) instead of a request.
The Python/NodeSDK will set the sid to "" by default. And as soon as they receive a UpdateRoomEvent, they will update the field manually.

The drawback I see from the Request is that the Python/NodeSDK must have called the GetSidRequest at least once to get the sid. (Which isn't always the case)

Reply:
alright, should be able to just remove all FFI additions, since `maybe_sid()` sends `None` if it's empty, and the SID update triggers on Room update regardless.

Reply:
now it should